pkg/os/smb/api.go

```diff
@@ -78,12 +77,26 @@ func (api *SmbAPI) NewSmbGlobalMapping(remotePath, username, password string) er
 		return fmt.Errorf("NewSmbGlobalMapping failed. output: %q, err: %v", string(output), err)
 	}
 	return nil
+	//TODO: move to use WMI when the credentials could be correctly handled
```

Comment:
this is not implemented yet?

Reply:
This WMI method works with anonymous SMB share but I met some blocker on credentials. 
https://pkg.go.dev/github.com/microsoft/wmi@v0.31.1/server2019/root/microsoft/windows/smb#MSFT_SmbGlobalMapping.Create

I'm still looking for the corresponding for `PSCredential` when WMI interface before moving this implementation to WMI.